net/rhp/session_test.go

```diff
@@ -14,10 +14,11 @@ import (
 
 	"lukechampine.com/frand"
 
+	"go.sia.tech/core/net/rpc"
```

Comment:
looks like this package wasn't added? I don't understand why this didn't cause a CI failure 🤔